.github/workflows/ci-aot.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish AOTCompatibility testApp
+
+on:
+  push:
+    branches: [ 'main*' ]
+    paths-ignore:
+    - '**.md'
+  pull_request:
+    branches: [ 'main*' ]
+    paths-ignore:
+    - '**.md'
+
+jobs:
+  aot-test:
+    strategy:
+      fail-fast: false  # ensures the entire test matrix is run, even if one permutation fails
+      matrix:
+        os: [ windows-latest ]
+        version: [ net6.0, net7.0 ]
+
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0 # fetching all
+
+    # powershell does not support AOT compiler yet
```

Comment:
We should figure out what the problem is - I don't believe that powershell doesn't support AOT compiler, the problem is very likely elsewhere.

Reply:
@eerhardt this is what I saw the other day from PS.
![image](https://github.com/open-telemetry/opentelemetry-dotnet/assets/16979322/0c5ded57-26a6-4b42-b53c-c4eacfe3d240)

@vitek-karas mentioned that it might be because of publishing the app twice in a row. But I'm not able to repro this behavior now.